.github/workflows/publish-cli-bins.yaml

```
@@ -22,75 +22,107 @@ jobs:
-            ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
+          fetch-depth: 0
 
       # Install dependencies
       - name: Install Dependencies
         run: go mod tidy
 
       - name: Build
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:6: Double quote to prevent globbing and word splitting

(shellcheck)